.circleci/push_doc.sh

```diff
@@ -62,9 +62,30 @@ touch .nojekyll
 # landing page on fairlearn.org
 if [ "$CIRCLE_BRANCH" = "main" ]
 then
-    echo "Copying landing_page.js into js directory"
+    echo "Copying all the newly generated files for the static landing page"
     ls $GENERATED_DOC_DIR/..
+
+    # js
+
```

Comment:
Nit: can we remove the black lines between description and command?

Reply:
Sure 💯